gfsphysics/CMakeLists.txt

```diff
@@ -12,8 +12,6 @@ if(32BIT)
 endif()
 
 set(CCPP_SOURCES
-    physics/namelist_soilveg.f
-    physics/set_soilveg.f
     physics/noahmp_tables.f90
```

Comment:
Looks like this is the last remaining file in gfsphysics. Is it used? Can we remove it?

Reply:
This will be done by Mike Barlage and Helin Wei as part of their work to move the Noah MP initialization into CCPP. For their work, the present PR is a requirement.

Reply:
Ok. Thanks. The exact same file is already in ccpp-physics. Which of those two is actually used (linked into the executable)? I guess it depends on which one is specified first in the link command.